pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.black]
 line-length = 79
 skip-string-normalization = true
-target-version = ["py35", "py36", "py37"]
+target-version = ["py36", "py37", "py38"]
```

Comment:
do we need a change in setup.py?

Reply:
setup.py already has `python_requires='>=3.6'`. do you have another property in mind that should be changed?

Reply:
```
    classifiers=[
        'Development Status :: 4 - Beta',
        'Operating System :: OS Independent',
        'Intended Audience :: Science/Research',
        'Programming Language :: Python',
        'Programming Language :: Python :: 3',
        'Topic :: Scientific/Engineering',
```
we usually list explicity versions here. but can be a followup (as i guess wasn't before)